### ISSUE TITLE
Include heading for docs/bad_dump.md

### ISSUE THREAD

**Issue by lizgarciao** [pulibrary/bibdata#2205]
### Steps to recreate
Visit docs/bad_dump.md in bibdata

### URL
[Bad dump document](https://github.com/pulibrary/bibdata/blob/main/docs/bad_dump.md)

### Issues
Does not have a heading

### Severity
N/A

### WCAG Conformance level (A, AA, AAA)
N/A

### User impact
Users that use screen readers cannot skip to the main content by using the navigation headings menu (in Voice Over) since there is no heading for this document.

### Recommended fix
Include heading

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave

### Notes
